### PR TITLE
docs(public-docsite-v9): embed fluentui-contrib docsite

### PR DESCRIPTION
**Pull request by dmytrokirpa**
## Previous behavior

Locating the [fluentui-contrib](https://github.com/microsoft/fluentui-contrib) repository has been challenging, making it difficult for developers to utilize the components available there.

## New behavior

A new section on the documentation site now presents a list of contributor packages along with their Storybook documentation.

Preview: https://fluentuipr.z22.web.core.windows.net/pull/33108/public-docsite-v9/storybook/index.html?path=/docs/contrib_welcome--docs

image

### Notes:

The documentation for the packages is currently basic. It would be beneficial if the owners of these packages could consider updating the documentation to reflect more detailed information.

## Related Issue(s)

- Partially implements https://github.com/microsoft/fluentui-contrib/issues/175

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Review by tudorpopams — APPROVED** [review]

File: apps/public-docsite-v9/.storybook/main.js [hunk 1]
```diff
@@ -40,4 +40,12 @@ module.exports = /** @type {Omit<import('../../../.storybook/main'), 'typescript
 
     return localConfig;
   },
+  refs: {
```
Comment (Hotell) [inline apps/public-docsite-v9/.storybook/main.js:43]: asked async but reposting for better visibility: will this impact our sb docs load performance on local machine ? ty

Comment (dmytrokirpa) [reply apps/public-docsite-v9/.storybook/main.js:43]: I haven't observed any changes in load performance, likely due to the fact that it's being lazy-loaded in the browser.

**Without refs:**
Screenshot 2024-11-01 at 13 16 43


**With refs:**
Screenshot 2024-11-01 at 13 15 54

File: apps/public-docsite-v9/.storybook/manager-head.html [hunk 2]
```diff
@@ -108,7 +108,8 @@
     color: #ffffff !important;
   }
 
-  .sidebar-subheading {
+  .sidebar-subheading,
+  button[data-action='collapse-ref'] {
```
Comment (dmytrokirpa) [inline apps/public-docsite-v9/.storybook/manager-head.html:112]: To ensure the styles of the `Contributors Packages` menu item are consistent with the other items in the sidebar, the `button[data-action='collapse-ref']` is the best selector I could identify.

Screenshot 2024-11-01 at 13 20 38

File: apps/public-docsite-v9/.storybook/main.js (hunk 1 above)
Comment (Hotell) [reply apps/public-docsite-v9/.storybook/main.js:43]: awesome , ty for double check !

File: apps/public-docsite-v9/.storybook/manager-head.html (hunk 2 above)
Comment (Hotell) [reply apps/public-docsite-v9/.storybook/manager-head.html:112]: can we add this intel to the source via comment ? ty

**Pull request merged**